oojs/application.js

```diff
@@ -1,13 +1,16 @@
-$(document).ready(function() {
-  $('#roller button.add').on('click', function() {
-    console.log("WAT")
-    $('.dice').append('<div class="die">0</div>');
-  });
+window.onload = function() {
+  document.querySelector('#roller button.add').onclick = function() {
```

Comment:
You should definitely move these into their own functions instead of keeping them anonymous.